sqs/variables.tf

```diff
@@ -29,4 +29,14 @@ variable "apply_resource" {
 variable "sqs_policy" {
   description = "allows a custom SQS policy to be set"
   default     = "default"
+}
+
+variable "dead_letter_queues" {
+  description = "The dead letter queue for failed messages to be sent to"
```

Comment:
Should the description read "... queue**s** for failed ..." as it is an array

Reply:
It shouldn't be an array. I was originally trying to use a dynamic block which takes an array but that didn't work. You can only have one dlq for an sqs queue so this makes more sense as a string. I've changed the variable and changed the name.